Cargo.toml

```diff
@@ -32,32 +32,202 @@ exclude = [
     "forc-test/test_data"
 ]
 
+[workspace.package]
+edition = "2021"
+version = "0.63.4"
```

Comment:
Just a not: although currently every package in the sway repo is on the same version, there have been talks of doing more granular releases where only the packages that changed get a version bump & release.

We can always change it back later, and for now this will be useful.

Reply:
Good catch, agree we are going to want to change this back once each package can have their own release cycles.

Reply:
Yep, sounds good. As I said in the description, probably best that for now every default to the workspace version and then if/when things depart, they do so individually. It may end up and only a handful diverge at one time, so best to centralise everything at the moment.

Either way, this now gives us the best of both worlds :)